packages/node_modules/@ciscospark/react-component-adaptive-card/src/index.js

```diff
@@ -46,13 +90,15 @@ class AdaptiveCard extends React.Component {
 AdaptiveCard.propTypes = {
   cards: PropTypes.array,
   displayName: PropTypes.string,
-  sparkInstance: PropTypes.object
+  sparkInstance: PropTypes.object,
+  verb: PropTypes.string
 };
 
 AdaptiveCard.defaultProps = {
   cards: [],
```

Comment:
This component is supposed to represent one card, so why are you making this prop an array, if you also only look at the first element only?

Reply:
If we need to to render multiple cards then no need to change parent component props as it is representing array of cards already